dockerfiles/stages/1-build-deps

```diff
@@ -53,7 +53,6 @@ RUN apt-get update --quiet \
   libpq-dev \
   libprocps-dev \
```

Comment:
`libprocps8` -> `libproc2-0` in Noble. Hence I would assume `libprocps-dev` should go to bullseye/focal exclusive as well.